e2e/wdio/platform/tests/input-group.e2e-spec.ts

```diff
@@ -0,0 +1,146 @@
+        const compactHeight = webDriver.getElementSize(inputGroupPage.compactGroupInput) as WebdriverIO.SizeReturn;
+
+        expect(defaultHeight.height).toBeGreaterThan(compactHeight.height);
+    });
+
+    it('check have disabled attr assigned', () => {
+        webDriver.waitElementToBePresentInDOM(inputGroupPage.disabledInput);
+        webDriver.waitForDisplayed(inputGroupPage.disabledInput);
+
+        expect(webDriver.getAttributeByName(inputGroupPage.disabledInput, 'ng-reflect-is-disabled')).toBe('true');
+        expect(webDriver.isEnabled(inputGroupPage.disabledInput)).toBe(false);
+    });
+
+    it('check disabled attr has disabled button', () => {
+        webDriver.waitForDisplayed(inputGroupPage.disabledInput);
+        expect(webDriver.isElementClickable(inputGroupPage.disabledInputButton)).toBe(false);
+    });
+
+    it('with Form input have text and button addons', () => {
+        webDriver.waitForDisplayed(inputGroupPage.withFormInput);
+        webDriver.setValue(inputGroupPage.withFormInput, inputGroupTestData.numeric_value);
+
+        expect(webDriver.getText(inputGroupPage.withFormInputTextAddon)).toContain('$');
+        expect(webDriver.getText(inputGroupPage.withFormInputTextAddon, 1)).toContain('0.00');
+        expect(webDriver.getText(inputGroupPage.withFormInputButtonAddon)).toContain('Submit');
+        expect(webDriver.getValue(inputGroupPage.withFormInput)).toBe(inputGroupTestData.numeric_value);
+    });
+
+    it('with form input have error tooltip and visual que if empty', () => {
+        webDriver.waitForDisplayed(inputGroupPage.withFormInput);
+        webDriver.scrollIntoView(inputGroupPage.withFormInput);
+        webDriver.click(inputGroupPage.withFormInput);
+        webDriver.click(inputGroupPage.withFormInputButtonAddon);
+        webDriver.mouseHoverElement(inputGroupPage.withFormInput);
+
+        expect(webDriver.executeScriptBeforeTagAttr(inputGroupPage.withFormInputQuestionMark, 'content')).toBe('"?"');
+        expect(webDriver.executeScriptAfterTagAttr(inputGroupPage.withFormInputAsterixMark, 'content')).toBe('"*"');
+        // TODO: Uncomment after merge
+        /*        if (webDriver.isBrowser('Safari')) {
+            expect(webDriver.getText(inputGroupPage.withFormInputErrorTooltip)).toEqual('Value is required');
+            return;
+        }
+        console.log('Skip hover check for Safari');*/
+    });
+
+    it('with form input have info tooltip', () => {
+        // if(webDriver.isIEorSafari()) {
+            webDriver.waitForDisplayed(inputGroupPage.withFormInput);
+            webDriver.scrollIntoView(inputGroupPage.withFormInput);
+            webDriver.mouseHoverElement(inputGroupPage.withFormInputQuestionMark);
+
+            expect(webDriver.getText(inputGroupPage.withFormInputInfoTooltip)).toBe('This is tooltip to help');
+           // return;
+       // }
+    });
+
+    // TODO Unxit after merge
+    xit('should check RTL', () => {
```

Comment:
Why skipped?  I think you could add in the same manner as all the other tests, yes?